lib/logstash/outputs/loggly.rb

```diff
@@ -59,6 +59,12 @@ class LogStash::Outputs::Loggly < LogStash::Outputs::Base
   # https://www.loggly.com/docs/source-groups/
   config :tag, :validate => :string, :default => "logstash"
 
+  # Retry count
```

Comment:
Can you provide a more detailed explanation? To inform the user - what would cause a retry? What occurs if the retry count is exceeded for a given event?